atlite/cutout.py

```diff
@@ -150,7 +150,10 @@ def __init__(self, path, **cutoutparams):
 
         path = Path(path).with_suffix(".nc")
         chunks = cutoutparams.pop("chunks", {"time": 100})
-        storable_chunks = {f"chunksize_{k}": v for k, v in (chunks or {}).items()}
+        if isinstance(chunks, dict):
+            storable_chunks = {f"chunksize_{k}": v for k, v in (chunks or {}).items()}
```

Comment:
Is auto-chunking stable enough nowadays to become the default?

Reply:
I don't think we should set it as default yet. There might still be some problems with `atlite` functionalities like reprojecting, which do not fully work with chunked spatial dimensions.

Reply:
`(chunks or {})` is equivalent to `chunks` (since `chunks` is a dict)